CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+3.21.3:
+    - Guard against /sys/hypervisor/uuid not being readable (ENT-9931)
+    - Fixed lib/testing.cf tap output (CFE-4245)
+    - Removed jq dependency in lib/testing.cf and fixed tap output (CFE-4242)
```

Comment:
```suggestion
    - Removed jq dependency in lib/testing.cf and fixed tap output (CFE-4242, CFE-4245)
```

We can merge these together instead of repeating how tap output was fixed.